src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -6299,7 +6299,7 @@ Real64 CalcClearRoof(EnergyPlusData &state,
 
     Real64 Rf = RoughnessMultiplier[(int)RoughnessIndex];
     if (Rex > 0.1) { // avoid zero and crazy small denominators
-        Real64 tmp = std::log(1.0 + GrLn / pow_2(Rex));
+        Real64 tmp = std::log1p(GrLn / pow_2(Rex));
```

Comment:
Testing this suggestion:

[src/EnergyPlus/ConvectionCoefficients.cc:6302]:(style),[unpreciseMathCall],Expression 'log(1 + x)' can be replaced by 'log1p(x)' to avoid loss of precision.